src/goto-programs/remove_function_pointers.cpp

```diff
@@ -144,6 +144,33 @@ class remove_function_pointerst
     const goto_programt &goto_program,
     functionst &functions,
     const exprt &pointer);
+
+  /// From *fp() build the following sequence of instructions:
+  ///
+  /// if (fp=&f1) then goto loc1
+  /// if (fp=&f2) then goto loc2
+  /// ..
+  /// loc1: f1(); goto N+1;
+  /// loc2: f2(); goto N+1;
+  /// ..
+  ///
```

Comment:
Can we have a `locN+1` at the end for clarity?

Reply:
Done.